files/build_templates/instance/database.globalservice.j2

```diff
@@ -1,16 +1,27 @@
 [Unit]
```

Comment:
Can we keep old simple filename?
We could use symlink in git repo. both inside this folder and upper folder.

Reply:
Fixed as per comment.